### PR TITLE
MAGEDOC-3673/MC-3673

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request adds a release note describing MC-3673 to the 2.3.1 Release Notes. 

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.html
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.html

### CODE REVIEW

File: guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Magento Commerce 2.3.1 Release Notes
 ---
 
 
-*Release notes published on March 26, 2019 and last edited on July 31, 2019.*
+*Release notes published on March 26, 2019 and last edited on August 15, 2019.*
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md]:
```suggestion
*Release notes published on March 26, 2019 and last edited on August 16, 2019.*
```

**Comment by jfrontain** [comment]
running tests

File: guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md [hunk 2]
```diff
@@ -241,6 +241,12 @@ We've fixed hundreds of issues in the Magento 2.3.1 core code.
 
 <!--- MC-5620-->* The `bin/magento setup:upgrade --convert-old-scripts=1` command now supports the conversion of indexes and constraints.  
 
+<!--- MC-1364-->
+* The **Allow Dynamic Media URLs in Products and Categories** configuration setting, which was previously accessed from **Stores** > **Configuration** > **Catalog** > **Storefront**, has been removed. The **Use Static URLs for Media Content in WYSIWYG** setting (**Stores** > **Configuration** > **General** > **Content management** > **WYSIWYG Options**) now applies to any media URLs that are entered through the WYSIWYG editor.
+
+
```
Comment (keharper) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md]: Delete all these extra lines.

Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md]:
```suggestion
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md [hunk 3]
```diff
@@ -241,6 +241,12 @@ We've fixed hundreds of issues in the Magento 2.3.1 core code.
 
 <!--- MC-5620-->* The `bin/magento setup:upgrade --convert-old-scripts=1` command now supports the conversion of indexes and constraints.  
 
+<!--- MC-1364-->
+* The **Allow Dynamic Media URLs in Products and Categories** configuration setting, which was previously accessed from **Stores** > **Configuration** > **Catalog** > **Storefront**, has been removed. The **Use Static URLs for Media Content in WYSIWYG** setting (**Stores** > **Configuration** > **General** > **Content management** > **WYSIWYG Options**) now applies to any media URLs that are entered through the WYSIWYG editor.
+
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md]:
```suggestion
```

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5199) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5199) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by jfrontain** [comment]
running tests

**Pull request merged**